communities/limassol-cy/community-assesment.md

```diff
@@ -0,0 +1,50 @@
+## Assessing Your Community's Needs
+
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - High School Hackers in and around Limassol, Cyprus
+  
+### Size
+- Currently there's around 50 people in the community.
+- We'd like to see around 100 people until July. 
+- We had a sharp increase when we founded the community last April, and after the last event, in December 2016.
+
+### Resources
+- Currently we have our events, both the 24 hour hackathon and the 12 hour hackdays. 
+- Hackathon: Ninja Hacks - April & Local Hack Day  - Winter
+
+### Accessibility
+- Everything is fairly accsible, online and the in person workshops. 
+- Hackathons, Workshops are held on Saturdays. 
+
+### Interests
+- The community is made up of only first time hackers introduced to the community through our hackathons, they come from all backgrounds from economics to art.
+- It's completely open to any high schooler that is interested in creating and technology. 
```

Comment:
How do you reflect this open nature to the community? What does it look like from someone looking outside -> in?